src/main/kotlin/com/coffee/api/cafe/presentation/adapter/dto/response/DetailTagResponse.kt

```
@@ -0,0 +1,8 @@
+package com.coffee.api.cafe.presentation.adapter.dto.response
+
+class DetailTagResponse(
+    val id: String,
+    val name: String,
+    val imageUrl: String,
+) {
+}
```

detekt (1.23.7)

[warning] The class or object DetailTagResponse is empty.

(detekt.empty-blocks.EmptyClassBlock)